<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paint</class>
 <widget class="QWidget" name="Paint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paint</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>711</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="orangeButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Users/vikto/Downloads/orange.jpg</normaloff>../../../../Users/vikto/Downloads/orange.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>54</width>
     <height>54</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="purpleButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Users/vikto/Downloads/purple.jpg</normaloff>../../../../Users/vikto/Downloads/purple.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="blueButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Users/vikto/Downloads/blue.jpg</normaloff>../../../../Users/vikto/Downloads/blue.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>74</width>
     <height>74</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="lightBlueButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Users/vikto/Downloads/lblue.jpg</normaloff>../../../../Users/vikto/Downloads/lblue.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="greenButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Users/vikto/Downloads/green.jpg</normaloff>../../../../Users/vikto/Downloads/green.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="redButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Users/vikto/Downloads/red.jpg</normaloff>../../../../Users/vikto/Downloads/red.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>54</width>
     <height>52</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="yellowButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Users/vikto/Downloads/yellow.jpg</normaloff>../../../../Users/vikto/Downloads/yellow.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>74</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="brownButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Users/vikto/Downloads/brown.jpg</normaloff>../../../../Users/vikto/Downloads/brown.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>74</width>
     <height>74</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="blackButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Users/vikto/Downloads/black.jpg</normaloff>../../../../Users/vikto/Downloads/black.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="rubberButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Users/vikto/Downloads/rubber.png</normaloff>../../../../Users/vikto/Downloads/rubber.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
